openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.10/1-eventing-crds.yaml

```diff
@@ -2279,7 +2279,7 @@ spec:
     - knative
     - messaging
     shortNames:
-    - sub
+    - ksub
```

Comment:
Thanks for the interest in Openshift Serverless, but we can not do that he. We distribute the manifests from upstream and stay aligned with those.

This `shortName` is defined here:
https://github.com/knative/eventing/blob/main/config/core/resources/subscription.yaml#L247

Feel free to reach out to upstream knative/eventing project for discussing the name change

no, we can not do that.

Reply:
Ok that sounds reasonable. I will try upstream.